src/ConnectivityWizard/Endpoints/FinishEndpoint.ts

```diff
@@ -0,0 +1,18 @@
+import { Endpoint } from './Endpoint';
+import logger from '../../Utils/Logger';
+import Koa from 'koa';
+
+export class FinishEndpoint implements Endpoint {
+  public handle(ctx: Koa.Context): Promise<void> {
```

Comment:
This method should return promise according to its signature, but it doesn't
```suggestion
  public async handle(ctx: Koa.Context): Promise<void> {
```